src/openrct2/rct1/S4Importer.cpp

```diff
@@ -1174,7 +1174,7 @@ class S4Importer final : public IParkImporter
         dst->ride_subtype = ride->subtype;
 
         dst->vehicle_type = vehicleEntryIndex;
-        dst->is_child = src->is_child;
+        dst->is_child = src->type;
```

Comment:
Perhaps it's a good idea to also rename `is_child` to `type` on the regular sprite.

Reply:
I have already done that on my follow up PR along with a new getter method, IsChild().